### PR TITLE
HUB-963: Remove ida support email from public pages

### PR DESCRIPTION
**Pull request by CharlesIC**
Use the feedback form instead to let people contact us

- Remove most pages as we're no longer accepting new services
- Update sitemap
- Point to the feedback form instead of the email
- Remove the "Tried and trusted" paragraph from the Verify product page
- Update the support page

All changes applied as specified by Paola in the [product page doc](https://docs.google.com/document/d/1rkevXTbXpJEHNAjjtzYGCaUln0HXATqdeU6XagZchqA/edit#).

### CODE REVIEW

File: source/support.html.erb [hunk 1]
```diff
@@ -48,9 +48,7 @@ title: Support – GOV.UK Verify
             if it's a known issue with the platform, it will show there and we'll keep our users updated on how the incident is progressing
             </li>
             <li>
-              if there are no incidents, email
-              <a href="mailto:idasupport@digital.cabinet-office.gov.uk" class="govuk-link">idasupport@digital.cabinet-office.gov.uk</a>
-              with the details of the problem your service is experiencing
+              if there are no incidents, email our support team with the details of the problem your service is experiencing
```
Comment (galund) [inline source/support.html.erb:51]: This feels wrong: well-run services teams might have avoided writing down the email address, and will have referenced either this page, or 
a page that links to it, in their team manual. We should still make it easy for existing services to get in touch via email, without going via the end-user support team.

Comment (CharlesIC) [reply source/support.html.erb:51]: We can email all services with an FYI message to save our support email address somewhere. Leaving it on a public page will defeat the purpose of making these changes and allow people to continue emailing us with random queries and bloating our 2nd line.

Comment (CharlesIC) [reply source/support.html.erb:51]: Agreed to move this content to self-service and hide behind a log-in page

**Review by j-maynard — APPROVED** [review]
🦁

**Review by JohnWattsGDS — APPROVED** [review]

**Pull request merged**